test/support/schemas/starknet_api_openrpc.json

```diff
@@ -0,0 +1,1964 @@
+{
```

Comment:
What about methods from the [write api](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_write_api.json)?

Reply:
I'm not sure about keeping such files in the repo. Maybe there is another way of achieving that? Or do you think it is ok?

Reply:
I don't like keeping it in the repo either. This is a modified version of the RPC spec (some usages of `oneOf` replaced with `anyOf`, fixed some bugs), because the 0.2.1 release had some problems that interfered with the validation.

Ideally (once new rpc spec is released hopefully addressing this problem), we could use a git submodule instead.

Reply:
Yup, it will be the best solution. We should keep it in mind